spec/models/user_spec.rb

```diff
@@ -48,10 +48,11 @@
     expect { user.save! }.to raise_error(ActiveRecord::RecordInvalid)
   end
 
-  it "should not save user with password shorter than 8 characters" do
-    user = build(:user, password: "a" * 7)
-    expect { user.save! }.to raise_error(ActiveRecord::RecordInvalid)
-  end
+  #Not currently relevant since db only stores password hashes, checking should be done in user form.
```

Comment:
Delete then?

Reply:
Leaving the test implies we may need it, which it seems won't be the case.

Reply:
I'll take it out. Password length is checked in the form on account creation